tests/components/modbus/conftest.py

```diff
@@ -132,57 +142,23 @@ async def mock_modbus_fixture(
             }
         ]
     }
-    mock_pb = mock.AsyncMock()
-    mock_pb.close = mock.MagicMock()
+    now = dt_util.utcnow()
     with mock.patch(
-        "homeassistant.components.modbus.modbus.AsyncModbusTcpClient",
-        return_value=mock_pb,
+        "homeassistant.helpers.event.dt_util.utcnow",
+        return_value=now,
         autospec=True,
```

Comment:
What is this mock doing?

Reply:
mock_do_cycle, secures that the HA event engine runs a step, if you look at:

https://github.com/home-assistant/core/blob/764a0f29cc3f557d1365da08f6e6c24ffbd65b91/tests/components/modbus/test_sensor.py#L717

there you see how config_addon as well as this mock is used.

Without this mock, the setup would run, but not the tasks scheduled nor the first update run.

Reply:
No I mean, You're mocking `dt_util.utcnow` to return `dt_util.utcnow`...?

Reply:
Yes and no, I am mocking it to always deliver the same time.